source/clear-linux/guides/maintenance/cpu-performance.rst

```diff
@@ -0,0 +1,176 @@
+.. _cpu-performance:
+
+CPU Power and Performance
+#########################
+
+Modern x86 CPUs employ a number of features and technologies to balance
+performance, energy, and thermal efficiencies.
+
+By default, |CL| prioritizes maximum CPU performance with the philosophy that
+the faster programs finishes execution, the faster the CPU can return to a low
+energy idle state. It is important to understand and evaluate all of these
+technologies when troubleshooting or considering changing the defaults.
+
+
+.. contents:: :local:
+    :depth: 2
+
+
+CPU clock frequency scaling
+===========================
+
+The CPUFreq subsystem in Linux allows the OS to control C-states and P-states
+via CPU drivers and governors that provide algorithms defining how and when to
+enter these states. 
```

Comment:
Lacking any explanation of what C-states are. Which would really help to explain some of the notes.

Reply:
I think you need to explain the concepts at the top, then dive into the technical details. Now you're mixing it around.